src/js/components/__tests__/__snapshots__/README-test.js.snap

```diff
@@ -19120,17 +19237,7 @@ boolean
 
 **messages**
 
-Custom messages. Used for accessibility by screen readers. Defaults to \`{
-  \\"closeMenu\\": \\"close menu\\",
-  \\"fullScreen\\": \\"full screen\\",
-  \\"progressMeter\\": \\"video progress\\",
-  \\"scrubber\\": \\"scrubber\\",
-  \\"openMenu\\": \\"open menu\\",
-  \\"pauseButton\\": \\"pause\\",
-  \\"playButton\\": \\"play\\",
-  \\"volumeDown\\": \\"volume down\\",
-  \\"volumeUp\\": \\"volume up\\"
-}\`.
+Custom messages. Used for accessibility by screen readers.
```

Comment:
Why did we remove the default messages? Is it breaking backward compatibility? Should we add a reference to the new messages object?

Reply:
The components don't specify a default for their messages object anymore since the messages come from the overall grommet messages default. This doesn't break backward compatibility since if they don't specify a messages property to the component they still get the same text. It just comes from a different place internally. The doc.js does still tell you what the default message text is. If you have some thoughts about pointing them toward grommet's overall messages that might help? It's more like the theme now really.